scipy/stats/tests/test_continuous_basic.py

```diff
@@ -6,6 +6,7 @@
 from scipy.integrate import IntegrationWarning
 
 from scipy import stats
+from scipy.stats import boost
```

Comment:
Something's going on here and I can't run `test_continuous_basic` with `runtests.py`. I can do it with `pytest` directly, but with `python runtests.py -v -t scipy/stats/tests/test_continuous_basic.py` I get:
```
_ ERROR collecting build/testenv/lib/python3.7/site-packages/scipy/stats/tests/test_continuous_basic.py _
ImportError while importing test module '/Users/matthaberland/Desktop/scipy/build/testenv/lib/python3.7/site-packages/scipy/stats/tests/test_continuous_basic.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
scipy/stats/tests/test_continuous_basic.py:9: in <module>
    from scipy.stats import boost
scipy/stats/boost/__init__.py:1: in <module>
    from ._stats.bernoulli import bernoulli
E   ModuleNotFoundError: No module named 'scipy.stats.boost._stats'
```

Reply:
This was due to a missing subpackage in a setup.py file, should be working now